examples/basics/custom_embeddings_sdk.ipynb

```diff
@@ -223,7 +223,7 @@
       "metadata": {},
       "source": [
         "# Delete a custom embedding\n",
-        "embedding.delete()"
+        "#embedding.delete()"
```

Comment:
Is this intentional?

Reply:
Yes, I usually comment out any deletions just in case customers run the notebook from start to end without reading all the code